util/qmk_install.sh

```diff
@@ -10,7 +10,7 @@ case $(uname) in
 	Linux)
 		exec "${util_dir}/linux_install.sh"
 	;;
-	MSYS_NT*)
+	M*_NT*)
```

Comment:
Sorry to contradict what @drashna said, but I've been busy and this was just brought to my attention. Rather than have a very broad wildcard can this be changed to `MSYS_NT*|MINGG_64*)`?

Reply:
That's why I tagged you in it. :)
Because I wasn't really fond of the wildcards myself but wasn't sure about how to handle this.

Reply:
Sorry for being slow on this! I just saw the message. Updated!